src/collection.js

```diff
@@ -900,6 +908,17 @@ export default class Collection {
   }
 
   /**
+   * Return a copy of the specified record without the local fields.
+   *
+   * @param  {Object} record  A record with potential local fields.
+   * @return {Object}
+   */
+  cleanLocalFields(record) {
```

Comment:
Wondering out loud, wouldn't we want an option to be added to `get()`, `list()` and possibly other methods so we could retrieve cleaned records? eg. `excludeLocalFields` or eq.